states.go

```diff
@@ -52,6 +52,8 @@ const (
 	// UpdateHubStateWaitingForReboot is set when the agent is waiting
 	// for reboot
 	UpdateHubStateWaitingForReboot
+	// UpdateHubStateFinal is set when the daemon is about to quit
+	UpdateHubStateFinal
```

Comment:
Which one do you think is better?

StateFinal
StateExit
StateDone
StateQuit